assets/src/application/ui/display/EmptyState.tsx

```diff
@@ -2,31 +2,31 @@
  * External dependencies
  */
 import * as React from 'react';
+import { __ } from '@wordpress/i18n';
 import { Empty } from 'antd';
 
 interface EmptyStateProps {
-	children: React.ReactChild;
+	children?: React.ReactNode;
+	className?: string;
 	description: string;
-	title: string;
+	title?: string;
 }
 
-const EmptyState = (props: EmptyStateProps) => {
-	const { children, description, title } = props;
-
+const EmptyState = ({ className, children, description, title }: EmptyStateProps) => {
 	return (
-		<>
+		<div className={className}>
 			<Empty
 				description={
 					<>
-						{title}
-						<span>{description}</span>
+						{title && __(title)}
+						<span>{__(description)}</span>
 					</>
 				}
 				image={Empty.PRESENTED_IMAGE_SIMPLE}
 			/>
 
-			{children}
-		</>
+			{children && children}
```

Comment:
ok I guess this IS ok...

how does this not just print that variable twice? or not at all?

like how come it will print the variable the second time it is encountered and not the first time?
I'm guessing this is just shorthand for a ternary right? Just not something I've ever seen documented anywhere so never fully grokked the underlying mechanics.

what would happen if you did `{children && children && children}` ???

not saying you need to change this but man... JS is crazy!!!

Reply:
> how does this not just print that variable twice? or not at all?
cuz `{ undefined && smth }` is the same as `{ false && smth }` which won't return anything

Reply:
my understanding is that it's not a shorthand for ternary operator
this is from MDN:
![image](https://user-images.githubusercontent.com/1477453/74269936-001f7380-4d13-11ea-8318-9d5b91ced1d8.png)

*** 
it doesn't have direct link :(
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Logical_Operators

Reply:
BTW, I wasn't aware of this part yet `if expr is a function call, the calling never takes place`

Reply:
last comment: before applying that logic there I had checked first things on antd side of things to make sure I am not goofing smth, so they have that prop optional:
https://github.com/ant-design/ant-design/blob/65eb713fe897f65bbf187e215c664e5f9b468252/components/empty/index.tsx#L22
***
and this is why they also applied that short-circuit eval as I did: https://github.com/ant-design/ant-design/blob/65eb713fe897f65bbf187e215c664e5f9b468252/components/empty/index.tsx#L68

Reply:
> cuz { undefined && smth } is the same as { false && smth } which won't return anything

right.. i get that part, but my brain wants to think that `{ smth && smth }` would return `smth smth` resulting in a duplicate, but that's obviously not the case.